syntax = "proto3";

package test.integration.filters;

message ExpectSpan {
  string operation_name = 1;
  bool sampled = 2;
  bool context_injected = 3;
  map<string, string> tags = 4;
}

message TracerTestConfig {
  repeated ExpectSpan expect_spans = 1;
}
